### ISSUE TITLE
Correct answer view

### ISSUE THREAD

**Issue by PodviaznikovPavlo** [hashobject/quizzy#19]

**Comment by PodviaznikovPavlo** [comment]
@podviaznikov 

i have done that task (the answer that begins with lower case) Cat = cat now.

**Comment by podviaznikov** [comment]
Ok, where is pull request for this task with the code?

**Comment by podviaznikov** [comment]
what is this issue? Still active or should be closed?

**Comment by PodviaznikovPavlo** [comment]
@podviaznikov 
yes, here we compare answer with user answer. all answers must be lower cased. it is done.
